### PR TITLE
Feat: Added Search, sort, upload video feature

### PR DESCRIPTION
**Pull request by vinc3nati**
- search page
- sort on basis of timestamp
- upload video

image

image

image

image

image

### CODE REVIEW

**Review by Aman-Jain1996 — APPROVED** [review]
Clean code , working functionality and good to merge

**Review by ramalingaa — APPROVED** [review]
Clean code, logic working , good to merge

File: src/components/Navbar/navbar.css [hunk 1]
```diff
@@ -9,30 +9,37 @@
...
-  .search-bar input[type="search"] {
+  .search-bar input {
     background-color: var(--color-background);
     color: var(--color-foreground);
   }
   
   .search-list {
     position: fixed;
-    background: var(--color-tertiary-dark) !important;
+    background: var(--color-background) !important;
```
Comment (OmKakatkar) [inline src/components/Navbar/navbar.css:19]: Try not to use !important

**Review by OmKakatkar — APPROVED** [review]
Overall code looks clean.
Like the folder structure

**Pull request merged**